impact.html

```diff
@@ -45,60 +45,89 @@ <h5><span class="w3-opacity">DATE</span></h5>
 
         <div class="w3-justify">
           <img src="https://germini.info/wp-content/uploads/2016/12/JavaScript-if-else.jpg" alt="Girl Hat" style="width:100%" class="w3-padding-12">
-          <p><strong>Traditional taxi- Cab Operators and drivers:/strong></p><br><br>
+          <p><h1>Who are affected in the Disruption</h1></p><br>
+        
+          <p><h2>Traditional taxi- Cab Operators and drivers</h2</p><br>
+        
           <p>Essentially ride sharing is wiping out the taxi cab industry and particularly the operators and the drivers of the traditional taxi company.
              It forces them to either update and compete with the new trend. Unlike the regular taxis, ride sharing innovation is opening up a new market with its various forays, which the traditional cab company cannot cope up with.
              At a certain level that regular taxis cannot offer service like, dabbling in chopper rides, food delivery and more. This innovation allows the ride sharing company to grow and break the stranglehold taxis had on consumers.
              According to (limpin 2018), drivers chose to be in the ride sharing industry instead of a traditional one because of the financial benefits they enjoy from the service.</p></br>
```

Comment:
Please remove all the < br> after every headers and paragraphs since those commands already have breaks.